hack/lib/protoc.sh

```diff
@@ -62,7 +62,7 @@ function kube::protoc::protoc() {
   PATH="${gogopath}:${PATH}" protoc \
     --proto_path="${package}" \
     --proto_path="${KUBE_ROOT}/vendor" \
-    --gogo_out=plugins=grpc:${package} ${package}/api.proto
+    --gogo_out=plugins=grpc:"${package}" "${package}"/api.proto
```

Comment:
In general prefer quoting the whole word, not just the var

Reply:
@fejta Thanks for your feedback! I've updated the code.